packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -246,7 +246,9 @@ export class SignIn extends BaseResource implements SignInResource {
       this.firstFactorVerification = new Verification(data.first_factor_verification);
       this.secondFactorVerification = new Verification(data.second_factor_verification);
       this.createdSessionId = data.created_session_id;
-      this.userData = deepSnakeToCamel(data.user_data) as UserData;
+      this.userData = data.user_data
+        ? { ...(deepSnakeToCamel(data.user_data) as UserData), experimental_imageUrl: data.user_data.image_url }
+        : {};
```

Comment:
If `data.user_data` is missing, it would keep `undefined`/`null` value. Now, it sets it to `{}`. Is this intentional? Will it make a difference?

Reply:
It's not nullable in our types and we were just casting it to the type here. My guess is it's not going to cause any problems. The way it was before is more worrisome.